packages/terra-date-time-picker/src/DateTimePicker.jsx

```diff
@@ -3,7 +3,7 @@ import { injectIntl, intlShape } from 'react-intl';
 import PropTypes from 'prop-types';
 import classNames from 'classnames/bind';
 import ThemeContext from 'terra-theme-context';
-import DatePicker from 'terra-date-picker';
+import DatePicker from 'terra-date-picker/src/DatePicker';
```

Comment:
```suggestion
import DatePicker from 'terra-date-picker';
```

Reply:
Removed it here https://github.com/cerner/terra-framework/pull/1224/commits/22d489afc2983bcb40c258881a6ed2beab768874